zipkin-collector-service/build.gradle

```diff
@@ -1,6 +1,13 @@
 apply plugin: 'application'
 mainClassName = 'com.twitter.zipkin.collector.Main'
 
+run {
+    // When configKey is dev, collector and query share ../zipkin.db
+    workingDir rootProject.projectDir
```

Comment:
ps. this is the 'ah-hah!' which was screwing up collector vs query. Basically each was creating their own DB files.